### PR TITLE
perl: adds Net::Netmask and String::Random perl modules

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: pkgs/top-level/perl-packages.nix [hunk 1]
```diff
@@ -6945,6 +6945,14 @@ let self = _self // overrides; _self = with self; {
     propagatedBuildInputs = [ pkgs.pkgconfig pkgs.dbus XMLTwig ];
   };
 
+  NetNetmask = buildPerlPackage {
```
Comment (edwtjo) [inline pkgs/top-level/perl-packages.nix]: missing meta

File: pkgs/top-level/perl-packages.nix [hunk 2]
```diff
@@ -6945,6 +6945,14 @@ let self = _self // overrides; _self = with self; {
...
+    name = "Net-Netmask-1.9021";
+    src = fetchurl {
+      url = mirror://cpan/authors/id/N/NL/NLNETLABS/Net-Netmask-1.9021.tar.gz;
```
Comment (edwtjo) [inline pkgs/top-level/perl-packages.nix]: you can reuse ${name} with a rec

File: pkgs/top-level/perl-packages.nix [hunk 3]
```diff
@@ -8461,6 +8469,14 @@ let self = _self // overrides; _self = with self; {
     };
   };
 
+  StringRandom = buildPerlPackage {
```
Comment (edwtjo) [inline pkgs/top-level/perl-packages.nix]: missing meta

File: pkgs/top-level/perl-packages.nix [hunk 4]
```diff
@@ -8461,6 +8469,14 @@ let self = _self // overrides; _self = with self; {
...
+    name = "String-Random-0.20";
+    src = fetchurl {
+      url = mirror://cpan/authors/id/R/RO/ROSCH/String-Random-0.20.tar.gz;
```
Comment (edwtjo) [inline pkgs/top-level/perl-packages.nix]: same name

**Comment by domenkozar** [comment]
ping

**Comment by rycee** [comment]
@arno01, are you still interested in getting the String::Random and Net::Netmask Perl packages into Nixpkgs? Otherwise I'm inclined to close this ticket without adding them.

**Comment by zimbatm** [comment]
Closing as abandoned. Please let me know if this was a mistake !

**Pull request closed without merging**